test/test_tutorial.cpp

```diff
@@ -0,0 +1,173 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <fusion.h>
+#include <inlining.h>
+#include <ir/utils.h>
+#include <ops/alias.h>
+#include <ops/arith.h>
+#include <ops/utils.h>
+#include <test/utils.h>
+#include <test/validator.h>
+#include <type.h>
+
+namespace nvfuser {
+
+class Tutorial : public NVFuserTest {
+ protected:
+  static void SetUpTestSuite() {
```

Comment:
Where does this SetUpTestSuite get called to enable verbose?

Reply:
It should automatically called one.

Reply:
Oh, I see. It is static function.

Reply:
Yeah, I basically just copied the code from here https://github.com/google/googletest/blob/main/docs/advanced.md#sharing-resources-between-tests-in-the-same-test-suite